src/Infrastructure/Persistence/Doctrine/Repository/Regulation/LocationRepository.php

```diff
@@ -27,4 +32,123 @@ public function delete(Location $location): void
     {
         $this->getEntityManager()->remove($location);
     }
+
+    public function findOneByUuid(string $uuid): ?Location
+    {
+        return $this->createQueryBuilder('l')
+            ->addSelect('m')
+            ->where('l.uuid = :uuid')
+            ->innerJoin('l.measure', 'm')
+            ->setParameter('uuid', $uuid)
+            ->getQuery()
+            ->getOneOrNullResult()
+        ;
+    }
+
+    public function findAllForMapAsGeoJSON(
+        bool $permanentRegulationsOnly = false,
+        bool $temporaryRegulationsOnly = false,
+        bool $includeUpcomingRegulations = false,
+        bool $includePastRegulations = false,
+    ): string {
+        $query = $this->createQueryBuilder('l')
+            ->select('l.geometry, m.type, l.uuid')
+            ->innerJoin('l.measure', 'm')
+            ->innerJoin('m.regulationOrder', 'ro')
+            ->innerJoin('ro.regulationOrderRecord', 'roc')
+            ->where('roc.status = :status')
+            ->setParameters([
+                'status' => RegulationOrderRecordStatusEnum::PUBLISHED,
+                // 'now' => $this->dateUtils->getNow()->format('Y-m-d'),
+            ]);
+        /*
+                if ($permanentRegulationsOnly) {
+                    $query->andWhere('ro.endDate IS NULL');
+                } elseif ($temporaryRegulationsOnly) {
+                    $query->andWhere('ro.endDate IS NOT NULL');
+                }
+
+                if (!$includeUpcomingRegulations && !$includePastRegulations) {
+                    $query->andWhere('(ro.endDate >= :now OR ro.endDate IS NULL) AND ro.startDate <= :now');
+                }
+
+                if ($includeUpcomingRegulations) {
+                    $query->andWhere('ro.startDate > :now');
+                }
+
+                if ($includePastRegulations) {
+                    $query->andWhere('ro.endDate < :now');
+                }
+
+
+        */
+        $test = [];
+
+        $results = $query->getQuery()->getResult();
+        foreach ($results as $result) {
+            $test[] = [
+                'type' => 'Feature',
+                'geometry' => json_decode($result['geometry']),
+                'properties' => [
+                    'location_uuid' => $result['uuid'],
+                    'measure_type' => $result['type'],
+                ],
+            ];
+        }
+
+        return json_encode($test);
```

Comment:
```suggestion
        $features = [];

        $results = $query->getQuery()->getResult();
        foreach ($results as $result) {
            $features[] = [
                'type' => 'Feature',
                'geometry' => json_decode($result['geometry']),
                'properties' => [
                    'location_uuid' => $result['uuid'],
                    'measure_type' => $result['type'],
                ],
            ];
        }

```suggestion
        return json_encode([
            'type' => 'FeatureCollection',
            'features' => $features,
        ]);
```